gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-2-config.component.ts

```diff
@@ -210,8 +194,8 @@ export class Step2Entrypoints2ConfigComponent implements OnInit, OnDestroy {
         break;
       case 'NATIVE':
         this.stepService.goToNextStep({
-          groupNumber: 5,
-          component: Step5SummaryComponent,
+          groupNumber: 3,
```

Comment:
Up to you, but you can combine PROXY and NATIVE cases here.

Reply:
Ho yes